hortacloud-deployment/README.md

```diff
@@ -0,0 +1,15 @@
+# hortacloud-deployment
+
+Dockerfile to build an image for deployment of hortacloud
+
+## Usage
+Refer to hortacloud [deployment instruction](https://hortacloud.janelia.org/docs/administration/aws/deployment/) for detailed steps
+### Container setup
+- Create a **.env** file that contains all required environment variables.
+- Run `docker build .` to build a docker image.
```

Comment:
You can build it with a specific tag instead of looking for the image id after the fact.